src/utils/audio-manager.js

```diff
@@ -0,0 +1,34 @@
+import { func } from 'prop-types';
+
+class AudioManager {
+  constructor(url) {
+    this.setNewStation(url);
+  }
+
+  play() {
+    this.audio.play();
+  }
+
+  pause() {
+    this.audio.pause();
+  }
+
+  select(url) {
+    this.audio.pause();
+    this.audio.currentTime = 0;
+    this.setNewStation(url);
+    this.play();
+  }
+
+  setNewStation(url) {
+    this.audio = new Audio(url);
```

Comment:
There is no need to create a new instance on every station change. You can just use:

```js
this.audio.setAttribute('src', url);
```

`this.audio` as an `Audio` instance could be created in the constructor once.

Reply:
that's true!